internal/sdk/cloudian/qos.go

```diff
@@ -0,0 +1,132 @@
+package cloudian
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+)
+
+// QualityOfService configures data limits for a Group or User.
+type QualityOfService struct {
+	// Soft is the soft (warning) limit.
+	Soft QualityOfServiceLimits
```

Comment:
Is `Warning` more informative than `Soft`?

Reply:
I'm ok with Warning, how bout you @erikgb ?

Reply:
I am ok with either. Let's go for `Warning`.